.env.template

```diff
@@ -1,5 +1,5 @@
 # DB Connection string
-SQLALCHEMY_URL=postgresql://USER_HERE:PASSWORD_HERE@db:5432/imap
+SQLALCHEMY_URL=postgresql+psycopg://USER_HERE:PASSWORD_HERE@db:5432/imap
```

Comment:
Just for my own understanding, what does this `psycopg` do?

Reply:
The scarily named Psycho PG is the postgres driver for python. This change is needed by the update from v2 to v3 - it seems to insist on the driver name being in the connection string now.